app/api/helpers/order.py

```diff
@@ -134,7 +135,16 @@ def create_onsite_attendees_for_order(data):
     del data['on_site_tickets']
 
 
-def calculate_order_amount(tickets, discount_code):
+def calculate_order_amount(data):
```

Comment:
Don't change the function signature, it was fine. This should take tickets and discount code only.

Reply:
Create a wrapper function if you want to